test/cypress/integration/widgets/service_button_spec.ts

```diff
@@ -35,5 +40,14 @@ describe('Service Button widget', () => {
             cy.setBooleanConfigurationField(widgetId, basicButtonToggleName, true);
             cy.get('button').should('have.class', 'basic');
         });
+
+        it('tab activated on blueprint marketplace page by default', () => {
```

Comment:
Optional:

Probably 
```suggestion
        it('default marketplace tab', () => {
```
would be enough.